packages/xo-web/src/common/xo/index.js

```diff
@@ -1010,14 +1023,32 @@ export const startVms = vms =>
       )
     )
 
-    if (forbiddenStart.length === 0) {
+    if (forbiddenStart.length === 0 && vmsWithduplicatedMacAddresses.length === 0) {
       if (nErrors === 0) {
         return
       }
 
       return error(_('failedVmsErrorTitle'), _('failedVmsErrorMessage', { nVms: nErrors }))
     }
 
+    if (vmsWithduplicatedMacAddresses.length > 0) {
+      // Retry without checking MAC addresses
+      try {
+        await confirm({
+          title: _('forceStartVm'),
+          body: _('vmsWithDuplicatedMacAddressesMessage', { nVms: vmsWithduplicatedMacAddresses.length }),
+        })
+        await Promise.all(map(vmsWithduplicatedMacAddresses, id => _startVm(id, undefined, true, true)))
+      } catch (error) {
+        // Nothing to do
+      }
+    }
+
+    // Clone or retry with force
+    if (forbiddenStart.length === 0) {
```

Comment:
I think it would be easier to read if this `if` wrapped the corresponding piece of code, like the first one because it would be more symmetrical:
```js
if (!condition1 && !condition2) {
  return
}

if (condition1) {
  // do this
}

if (condition2) {
  // do that
}
```
What do you think?